build.gradle

```diff
@@ -152,3 +178,70 @@ task releaseConnector {
         }
     }
 }
+
+// Daily build configurations
+def packageOrg = "ballerinax"
+def platform = "java11"
+def artifactCacheParent = file("${project.projectDir}/build/cache_parent/")
+
+def packageName = "";
+def packageVersion = "";
```

Comment:
If these are only used within copyDistribution task, we can define these within that task.